environmentmodel/environment-model_shacl.ttl

```diff
@@ -0,0 +1,288 @@
+            sh:description "Ellipsodial height or orthometric height"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of heightSystem failed!"@en ;
+            sh:name "heightSystem"@en ;
+            sh:order 1 ;
+            sh:path environmentmodel:heightSystem ] ;
+    sh:targetClass environmentmodel:GeodeticReferenceSystem .
+
+environmentmodel:BoundingBoxShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of yMin failed!"@en ;
+            sh:name "yMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environmentmodel:yMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of yMax failed!"@en ;
+            sh:name "yMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 3 ;
+            sh:path environmentmodel:yMax ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMin failed!"@en ;
+            sh:name "xMin"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environmentmodel:xMin ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of xMax failed!"@en ;
+            sh:name "xMax"@en ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path environmentmodel:xMax ] ;
+    sh:targetClass environmentmodel:BoundingBox .
+
+environmentmodel:Coordinate2DShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:float ;
+            sh:message "Validation of x failed!"@en ;
+            sh:name "x"@en ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path environmentmodel:x ],
+        [ sh:datatype xsd:float ;
+            sh:message "Validation of y failed!"@en ;
+            sh:name "y"@en ;
+            sh:maxCount 1 ;
+            sh:order 1 ;
+            sh:path environmentmodel:y ] ;
+    sh:targetClass environmentmodel:Coordinate2D .
+    
+environmentmodel:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "State of project area"@en ;
+            sh:message "Validation of state failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "state"@en ;
+            sh:order 1 ;
+            sh:path environmentmodel:state ],
+        [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "nan"@en ;
+            sh:message "Validation of city failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "city"@en ;
+            sh:order 3 ;
+            sh:path environmentmodel:city ],
+        [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "Region of project area"@en ;
+            sh:message "Validation of region failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "region"@en ;
+            sh:order 2 ;
+            sh:path environmentmodel:region ],
+        [ skos:example "Germany, GER" ;
```

Comment:
cant you use the address shape which is already in the gaia-x vocabulary?

<http://www.w3.org/2006/vcard/ns#Address>

if yes you can add it as a shape with following approach:
```
[ sh:minCount 1 ;
            sh:node <http://www.w3.org/2006/vcard/ns#Address> ;
            sh:path environmentmodel:address;
            sh:order 5 ] ;
```
Or you could use PostalAddressShape:
```
[ sh:minCount 1 ;
            sh:node gx:PostalAddressShape ;
            sh:path environmentmodel:address;
            sh:order 5 ] ;
```
But this has only one property for a  ISO 3166-2 based string.

Reply:
As far as I know, the idea here was to use the region output from the geodatabase. However, we can take another look at this in the data provider group.

@MircoNierenz @3Dbastian

Reply:
Opened a new issue:
https://github.com/GAIA-X4PLC-AAD/ontology-management-base/issues/37